ubuntu_image/parser.py

```diff
@@ -51,6 +51,12 @@ class FileSystemType(Enum):
     vfat = 'vfat'
 
 
+class StructureRole(Enum):
+    mbr = 'mbr'
+    systemboot = 'system-boot'
+    systemdata = 'system-data'
```

Comment:
I think the attribute names should be `system_boot` and `system_data`.  I know the other example does use `uboot` instead of `u_boot` but that's okay because "u" isn't itself a word.